06-auto-scaling/README.md

```diff
@@ -359,15 +405,21 @@ Update your stack.
 
 _Do you see more instances than the configured "desired capacity"?_
 
+> No
+
 ##### Question: Termination Order
 
 _If an instance is automatically terminated, which is it, the last one created
 or the first?_
 
+>The first one
```

Comment:
Correct, but this is also contingent on them being in the same AZ and Launch Config/same version of the Launch Template. It's also the oldest instance because it is the instance closest to the next billing hour. See: https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-instance-termination.html